interfaces/interfaces.go

```diff
@@ -2,7 +2,7 @@ package interfaces
 
 type IWritter interface {
 	WriteToDisk() error
+	CheckIfExists() error
```

Comment:
let us create a domain folder and underneath move the interfaces file  
also the folder and file objects move as domain objects

Reply:
addressed in fa38b125828f208631c482c74923ffb3fbe4f1da